extra/setup.py

```diff
@@ -46,7 +46,7 @@ def create_version_file():
 
 requirements = [
     'numpy==1.19.5',
-    'gluoncv>=0.9.2,<0.11.0',
+    'gluoncv>=0.9.4,<0.11.0',
```

Comment:
You'll want to rebase with mainline, I have updated the setup.py files to make it so we don't have to specify numpy , gluoncv, etc. versions in multiple different places.

Reply:
nevermind, was looking at old commit